commands/paste-emoji/command.toml

```diff
@@ -0,0 +1,16 @@
+name = "Paste Emoji"
+description = "Lets you search through emojis and paste it into your current applciation"
+endpointURL = "https://slapcommands.vercel.app/api/utils/emoji-paster"
```

Comment:
Let's update the input hint --

It says "Type to Search NPM Packages", but it should reference searching for emojis.

Reply:
<img width="786" alt="Screen Shot 2022-03-24 at 1 23 45 PM" src="https://user-images.githubusercontent.com/59267/160003815-b8dbdd15-be8b-48d9-af5e-649431cf52c1.png">

Also seem to be getting some dupes in the response --
<img width="786" alt="Screen Shot 2022-03-24 at 1 25 12 PM" src="https://user-images.githubusercontent.com/59267/160003864-ed4fbd42-a4b5-42dc-9cc8-4e7f23157e99.png">
o

Reply:
Oh, forgot about the placeholder, was using the code from the NPM search command and building upon that actually 😅

Regarding the duplicates, it seemed like my data source ([a file](https://unicode.org/Public/emoji/14.0/emoji-sequences.txt) from https://unicode.org) contained unqualified emojis. I have updated the data which now only have fully-qualified emojis (so no duplicated).

I have pushed these changes to the [Slapcommands repo](https://github.com/AnishDe12020/slapcommands) and it should work properly now

Reply:
Looks like there are still duplicates and sometimes there is no emoji (try searching for "hell"). Maybe it's because we can't render a specific type of emoji.
![CleanShot 2022-03-25 at 15 03 58](https://user-images.githubusercontent.com/57078134/160146944-ac0c9432-519f-4f51-b939-b940471d684b.png)
![CleanShot 2022-03-25 at 15 03 14](https://user-images.githubusercontent.com/57078134/160146947-7c904f2e-6b6f-4a2c-8c16-e7015357b7c8.png)

Reply:
Hey, I just checked and it seems I had forgotten to commit the new data. The duplicated should be gone.

Regarding the emojis not being rendered, these are all valid emojis and render in other applications (Tried Gnome Text Editor, Google Search, Discord) and it renders just fine. (Note: Discord uses Twemoji)

There are 2 options - A fix for the emoji rendering system by the Slapdash team or I can manually remove the ones that don't render (however that involves manually checking which all doesn't and there are too many)

I will address the other changes tomorrow (IST)